python/pyspark/pandas/indexes/base.py

```diff
@@ -2604,7 +2604,7 @@ def __xor__(self, other: "Index") -> "Index":
         return self.symmetric_difference(other)
 
     def __rxor__(self, other: Any) -> "Index":
-        return other.__xor__(self)
+        raise NotImplementedError("Index.__rxor__ is currently not supported.")
```

Comment:
I mean `return NotImplemented`, not `raise NotImplementedError( ... )`.
https://docs.python.org/3.9/library/constants.html#NotImplemented

Reply:
thank you, done